### PR TITLE
add a server-side-rendered path that does not require the network

### LINKED ISSUE #2269: Create a pure client side page that we can run benchmarks against.
In order to measure performance on Condenser we would like a page that has no external api calls (pure) or dependencies. It should be available as a URL on both development & production to be used by benchmarking & healthchecks. In this way any performance penalties or gains associated with any change to  condenser can be measured in isolation.

### PR DESCRIPTION
**Pull request by gl2748**
closes #2269 

adds an endpoint that can be served without any network requests, for testing or benchmarking.

see README additions for instructions.

### CODE REVIEW

File: stacktrace.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+echo "STARTING"
+
+sudo perf record -p $1 -F 100 -g -- sleep 60 &
```
Comment (sneak) [inline stacktrace.sh]: no code in this repo can presume upon or request superuser privileges

Comment (gl2748) [reply stacktrace.sh]: Sorry this shouldn't be in this pr.

Comment (gl2748) [reply stacktrace.sh]: Fixed https://github.com/steemit/condenser/pull/2318/commits/984415efbe8f3ad91bb3515655d8d05ad4e7d1bc

File: package.json [hunk 2]
```diff
@@ -18,15 +18,16 @@
...
+    "performance-test": "npm run start PERFORMANCE_TEST=true"
   },
   "author": "Steemit, Inc.",
   "license": "MIT",
   "dependencies": {
     "@steem/crypto-session": "git+https://github.com/steemit/crypto-session.git#83a90b319ce5bc6a70362d52a15a815de7e729bb",
     "@steemit/steem-js": "0.6.8",
     "assert": "^1.3.0",
-    "autoprefixer-loader": "^3.2.0",
+    "autoprefixer-loader": "^3.2.0",
```
Comment (bnchdrff) [inline package.json]: why whitespace

File: src/shared/UniversalRender.jsx [hunk 3]
```diff
@@ -35,6 +35,16 @@ import ScrollBehavior from 'scroll-behavior';
 
 import { api } from '@steemit/steem-js';
 
+let get_state_perf,
+    get_content_perf = false;
+if (process.env.OFFLINE_SSR_TEST) {
+    const testDataDir = process.env.OFFLINE_SSR_TEST_DATA_DIR || 'api_mockdata';
+    let uri = `${__dirname}/../../`;
+    //.default because require
```
Comment (bnchdrff) [inline src/shared/UniversalRender.jsx]: i should've removed this line

File: package.json (hunk 2 above)
Comment (gl2748) [reply package.json]: will fix

File: src/shared/UniversalRender.jsx (hunk 3 above)
Comment (gl2748) [reply src/shared/UniversalRender.jsx]: will fix

**Comment by gl2748** [comment]
@bnchdrff can you approve the rebase - #2400

**Comment by bnchdrff** [comment]
superseded by #2400

**Pull request closed without merging**